### ISSUE TITLE
Receiving deprecation warning in Rails 5.1.1 using `eav_hash_for` class method

### ISSUE THREAD

**Issue by cebartling** [iostat/eav_hashes#7]
I'm getting the following deprecation warning when using the `eav_hash_for` class method:

```
DEPRECATION WARNING: Passing a class to the `class_name` is deprecated and will raise an ArgumentError in Rails 5.2. It eagerloads more classes than necessary and potentially creates circular dependencies. Please pass the class name as a string:
``` 

The offending line seems to be line 20 of *lib/eav_hashes/activerecord_extension.rb*:

`has_many :#{options[:entry_assoc_name]}, class_name: #{options[:entry_class_name]}, foreign_key: "#{options[:parent_assoc_name]}_id", dependent: :delete_all`

**Comment by iostat** [comment]
Hi @cebartling!
Thanks for reporting this. I believe the fix is as simple as surrounding `#{options[:entry_class_name]}` with double quotes.
If you open a PR, I'll be happy to merge it and add you to the "special thanks" list, as well as push the fixed version to RubyGems. Alternatively, I could do the same within a few days.

Thanks!

**Comment by cebartling** [comment]
@iostat I will open a PR soon. We were just talking about contributing a fix to this. Thanks for getting back to me on this issue. You should see something in the next couple of hours.

**Comment by cebartling** [comment]
@iostat Any thoughts on the PR I opened? Anything I can do to get this merged and fixed for people using Rails 5.1?
